package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "node",
+  "version": "1.0.0",
+  "description": "> If you are following the HackYourFuture curriculum we recommend you to start with module 1: [HTML/CSS/GIT](https://github.com/HackYourFuture/HTML-CSS). To get a complete overview of the HackYourFuture curriculum first, click [here](https://github.com/HackYourFuture/curriculum).",
+  "main": "index.js",
```

Once it is grouped with `server.js` you can then specify here that the main file of your backend is `server.js` as such:

```suggestion
  "main": "server.js",
```

---

I get this